src/components/proxy/proxy-groups.tsx

```diff
@@ -597,15 +597,17 @@ export const ProxyGroups = (props: Props) => {
         initialScrollTop={scrollPositionRef.current[mode]}
         computeItemKey={(index) => renderList[index].key}
         itemContent={(index) => (
-          <ProxyRender
-            key={renderList[index].key}
-            item={renderList[index]}
-            indent={mode === "rule" || mode === "script"}
-            onLocation={handleLocation}
-            onCheckAll={handleCheckAll}
-            onHeadState={onHeadState}
-            onChangeProxy={handleChangeProxy}
-          />
+          <div style={{ paddingRight: 14, paddingLeft: 14 }}>
```

Comment:
```suggestion
          <div style={{ paddingRight: 14 }}>
```

左边间距可以不要，大概效果如图

<img width="1005" height="520" alt="Image" src="https://github.com/user-attachments/assets/f861269d-0ede-4f1e-8113-e019c90dbca2" />